powershell_completions_test.go

```diff
@@ -109,7 +109,7 @@ func TestPowerShellCompletion(t *testing.T) {
 	for _, tc := range tcs {
 		t.Run(tc.name, func(t *testing.T) {
 			buf := new(bytes.Buffer)
-			tc.root.GenPowerShellCompletion(buf)
+			er(tc.root.GenPowerShellCompletion(buf))
```

Comment:
Similar, since this is a test, we should probably do something like `assertNoErr(t, ...)`